pkg/configtx/policies.go

```diff
@@ -139,13 +137,15 @@ func (c *ConfigTx) AddApplicationOrgPolicy(orgName, modPolicy, policyName string
 
 // RemoveApplicationOrgPolicy removes an existing policy from an application organization.
 // The removed policy must exist.
```

Comment:
Might want to add the note about not erroring if it doesn't exist and remove this line instead (like all the other functions above)